Sources/MapboxCoreNavigation/EHorizon/ElectronicHorizonEdge.swift

```diff
@@ -1,48 +1,47 @@
 import Foundation
 import MapboxNavigationNative
 
-extension ElectronicHorizon {
+/**
+ An electronic horizon is a probable path (or paths) of a vehicle. This structure contains metadata about an edge in a routing graph. For example, an edge may represent a road segment between two intersections or between the two ends of a bridge. An edge may traverse multiple road objects, and a road object may be associated with multiple edges. The road network ahead of the user is represented as a tree of edges. Each intersection has outlet edges. In turn, each edge has a probability of transition to another edge, as well as details about the road segment that the edge traverses. You can use these details to influence application behavior based on predicted upcoming conditions.
+
+ During active turn-by-turn navigation, the user-selected route and its metadata influence the path of the electronic horizon determined by `RouteController`. During passive navigation (free-driving), no route is actively selected, so `PassiveLocationDataSource` will determine the most probable path from the vehicle’s current location. You can receive notifications about changes in the current state of the electronic horizon by observing the `Notification.Name.electronicHorizonDidUpdatePosition`, `Notification.Name.electronicHorizonDidEnterRoadObject`, and `Notification.Name.electronicHorizonDidExitRoadObject` notifications.
+
+ Use a `RoadGraph` object to get an edge with a given identifier.
+ */
```

Comment:
Since this is now the documentation comment for the edge type specifically, we shouldn’t bury the lede in terms of describing an edge. Let’s start off with the first paragraph of the old documentation, then follow with the rest.

Reply:
Rearranged some lines in this doc.